### PR TITLE
Split npm test:lint scripts.

### PR DESCRIPTION
**Pull request by XhmikosR**
This way we can run them individually too.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,11 @@
     "load-schedule": "curl -sS https://raw.githubusercontent.com/nodejs/Release/master/schedule.json -o source/schedule.json",
     "start": "npm run serve",
     "test": "npm run test:lint && npm run test:unit && npm run test:smoke",
-    "test:lint": "standard --fix && htmllint **/*.hbs && stylint layouts/css && markdownlint \"**/*.md\" -i \"node_modules/\"",
+    "htmllint": "htmllint **/*.hbs",
```
Comment (MylesBorins) [inline package.json:17]: I think a naming convention like this would be more consistent

```suggestion
    "test:lint:html": "htmllint **/*.hbs",
```

Comment (XhmikosR) [reply package.json:17]: TBH I prefer the shorter one, but it's your call. We can keep the current scheme for scripts that run other scripts.

Comment (MylesBorins) [reply package.json:17]: stick with the shorter one then... nbd

**Comment by XhmikosR** [comment]
Can we land this?

**Comment by Trott** [comment]
@nodejs/website This needs some reviews?

File: package.json (hunk 1 above)
Comment (alexandrtovmach) [reply package.json:17]: I agree with @MylesBorins ― consistency is more preferable.

Comment (XhmikosR) [reply package.json:17]: That's just extra characters to type without any real gain. Similarly with the other individual scripts, they are not all prefixed.

I strongly suggest that you guys reconsider. Typing less is better.

Comment (silverwind) [reply package.json:17]: While we're bikeshedding this, I suggest scheme `lint-xx` like we have in the Node.js repo and `lint` for the main task:

https://github.com/nodejs/node/blob/e4608381683de487c3c5844875e9adb692bb8725/Makefile#L1227

Comment (3imed-jaberi) [inline package.json:17]:
```suggestion
    "test:html-lint": "htmllint **/*.hbs",
```

File: package.json [hunk 2]
```diff
@@ -14,7 +14,11 @@
...
+    "markdownlint": "markdownlint \"**/*.md\" -i \"node_modules/\"",
```
Comment (3imed-jaberi) [inline package.json:18]:
```suggestion
    "test:markdown-lint": "markdownlint \"**/*.md\" -i \"node_modules/\"",
```

File: package.json [hunk 3]
```diff
@@ -14,7 +14,11 @@
...
+    "standard": "standard --fix",
```
Comment (3imed-jaberi) [inline package.json:19]:
```suggestion
    "test:standard-lint": "standard --fix",
```

File: package.json [hunk 4]
```diff
@@ -14,7 +14,11 @@
...
+    "stylint": "stylint layouts/css",
```
Comment (3imed-jaberi) [inline package.json:20]:
```suggestion
    "test:style-lint": "stylint layouts/css",
```

File: package.json [hunk 5]
```diff
@@ -14,7 +14,11 @@
...
+    "test:lint": "npm run standard && npm run htmllint && npm run stylint && npm run markdownlint",
```
Comment (3imed-jaberi) [inline package.json:21]:
```suggestion
    "test:lint": "npm run test:standard-lint && npm run test:html-lint && npm run test:style-lint && npm run test:markdown-lint",
```

**Review by 3imed-jaberi — COMMENTED** [review]
@XhmikosR , Thanks for your contribution ..
I agree with @MylesBorins here ..
So, I think the script key should be clearer more than short ( consistency ) ..
I have made some suggestions .. You can check 🚀 ..

**Comment by XhmikosR** [comment]
Sorry, I don't like this approach. You are welcome to make that change yourself. I don't see why I need to type the extra characters.

**Pull request closed without merging**